test/powershell/Modules/PSDiagnostics/PSDiagnostics.Tests.ps1

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe -Name "PSDiagnostics cmdlets tests" -Tag "CI","RequireAdminOnWindows" {
+    Context -Name "Test for Enable-PSTrace cmdlet" {
+        it -Name "Should enable Analytic logs for Microsoft-Windows-PowerShell." -Skip:(-not $IsWindows) {
```

Comment:
From what I've seen in the repo here with regards to tests, the prevailing style is generally the declarative, almost-DSL like syntax for Pester commands (most parameters not passed by name, all commands capitalised per usual PowerShell convention).

```suggestion
        It "Should enable Analytic logs for Microsoft-Windows-PowerShell." -Skip:(-not $IsWindows) {
```

Reply:
Even I didn't see -Name usage, but thought to follow the best practice. Changing this for uniformity makes sense.

Reply:
We should follow our common pattern and don't use `-Name` parameter.

Reply:
Thanks, already removed it.